tempest/api/workloadmgr/multiattach_volumes/test_multiattach_volumes.py

```diff
@@ -360,7 +368,7 @@ def test_01_multiattach_volumes(self):
             # Create workload with API
             try:
                 self.wid = self.workload_create([self.vm_id_1, self.vm_id_2],
-                                                tvaultconf.workload_type_id)
+                                                tvaultconf.workload_type_id, workload_cleanup=True)
```

Comment:
default cleanup value is True, no need to explicitly pass that parameter again.

Reply:
updated